### PR TITLE
Bugfix FXIOS-10551 Close Button Component Library Asset Updated

### PR DESCRIPTION
**Pull request by DanielDervishi**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-10551)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/23118)

## :bulb: Description

Updated the look of the close button to better account for accessibility. This should apply for all close button components. Changed after discussions with Jeff Guerrero (Designer)
Before: 
Close ButtonLight Mode BeforeClose Button Dark Mode Before

After:
Close Button Light Mode AfterClose Button Dark Mode After

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/Contents.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "crossCircleFillExtraLargeLight.pdf",
+      "filename" : "crossCircleFillExtraLarge 1.pdf",
```
Comment (issammani) [inline firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/Contents.json:4]: can we keep the original filenames ? I think `crossCircleFillExtraLarge(Light|Dark)` are more descriptive.

File: firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/crossCircleFillExtraLarge 1.pdf
Comment (issammani) [inline firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/crossCircleFillExtraLarge 1.pdf:1]: Both new files are "huge" compared to the originals. I am guessing these were exported as is from figma. Not sure about our process to compress these maybe @thatswinnie knows ?

**Review by issammani — CHANGES_REQUESTED** [review]
Hey Daniel, looks good. Just a couple of nits.

File: firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/Contents.json (hunk 1 above)
Comment (thatswinnie) [reply firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/Contents.json:4]: If these images need to be updated then the designers should update them in the Acorn library. We have a script that updates the images automatically.

File: firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/crossCircleFillExtraLarge 1.pdf
Comment (thatswinnie) [reply firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/crossCircleFillExtraLarge 1.pdf:1]: Thanks for the ping @issammani ! As described in the above comment: this is not the correct process to update these images. I think this PR should be closed.

**Review by thatswinnie — CHANGES_REQUESTED** [review]

**Comment by DanielDervishi** [comment]
Closed as per @thatswinnie's comments

**Pull request closed without merging**